engines/gob/save/saveload_v7.cpp

```diff
@@ -28,58 +28,1328 @@ namespace Gob {
 
 SaveLoad_v7::SaveFile SaveLoad_v7::_saveFiles[] = {
 	// Addy Junior Base
-	{"visage01.inf", kSaveModeSave, nullptr, "face"         }, // Child 01
```

Comment:
Isn't it breaking Addy Junior?

Reply:
"Addy Junior" is a synonym of "Adibou", it is the German name.